### PR TITLE
Bug 1907373: Tests for kube rebase to 1.20.0

### PR DESCRIPTION
**Pull request by tnozicka**
followup to the 1.20.0 rebase PR https://github.com/openshift/kubernetes/pull/471

/cc @sttts @soltysh

### CODE REVIEW

**Comment by tnozicka** [comment]
installer/infra
```
level=error msg=Error: error waiting for Route53 Hosted Zone (Z07148262WQW1FT0TA4FB) creation: timeout while waiting for state to become 'INSYNC' (last state: 'PENDING', timeout: 15m0s)
level=error
level=error msg=  on ../tmp/openshift-install-732050186/route53/base.tf line 18, in resource "aws_route53_zone" "int":
level=error msg=  18: resource "aws_route53_zone" "int" {
```
/retest

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Comment by tnozicka** [comment]
some images, but some infra
/retest

**Comment by tnozicka** [comment]
/retest
images is now mirrored

File: test/extended/util/annotate/rules.go [hunk 1]
```diff
@@ -34,6 +34,11 @@ var (
 			`should answer endpoint and wildcard queries for the cluster`, // currently not supported by dns operator https://github.com/openshift/cluster-dns-operator/issues/43
 			// https://bugzilla.redhat.com/show_bug.cgi?id=1902702 - [sig-auth][Feature:LDAP][Serial] ldap group sync can sync groups from ldap: oc cp over non-existing directory/file fails
 			`\[sig-auth\]\[Feature:LDAP\]\[Serial\] ldap group sync can sync groups from ldap`,
+
+			// https://bugzilla.redhat.com/show_bug.cgi?id=1906518
+			// It is already disabled in o/k but because openshift/csi suite prefixes the test name
+			// it no longer matches there.
+			`.*\[Feature:VolumeSnapshotDataSource\].*`,
```
Comment (tnozicka) [inline test/extended/util/annotate/rules.go:41]: the tests are still being run https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/25764/pull-ci-openshift-origin-master-e2e-gcp-csi/1339208077420793856

**Review by soltysh — APPROVED** [review]
/lgtm

**Comment by soltysh** [comment]
/hold
this failure:
```
fail [github.com/openshift/origin/test/extended/etcd/etcd_test_runner.go:39]: attempt to remove unknown resource settings.k8s.io/v1alpha1, Resource=podpresets
```
is real, this was removed in openshift/api, we probably should update etcd test accordingly.

**Comment by soltysh** [comment]
```
[sig-network] Networking Granular Checks: Services should function for service endpoints using hostNetwork [Suite:openshift/conformance/parallel] [Suite:k8s]
```
is a 1.20 test, but we don't allow using hostNetwork in OpenShift. I'd propose opening a BZ for networking team what they want to do with it. Seems like running this as privileged seems the easiest.

**Comment by tnozicka** [comment]
> is a 1.20 test, but we don't allow using hostNetwork in OpenShift.

we run all the kube tests as privileged, I'll look into it more closely and disable it if it won't be obvious

**Comment by tnozicka** [comment]
crio termination
/retest

**Comment by tnozicka** [comment]
/hold cancel
tests are green now

**Pull request merged**